src/graph_compute/numa_scheduling.h

```diff
@@ -183,14 +183,16 @@ static inline void createChunkData(vertex_t * const nodes,
   scheddata->numChunksPerWorker =
     (scheddata->cntChunks + NUMA_WORKERS - 1) / NUMA_WORKERS;
 
-  for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
+  cilk_for (vid_t i = 0; i < scheddata->cntChunks; ++i) {
```

Comment:
Sorry, I thought we resolved this.  I learned from Charles and Bradley that there is no problem using Cilk and pthreads in mutually exclusive portions of the code.  Technically, you're supposed to be able to use pthreads and Cilk concurrently, but that's a little dodgier.